services/config/knack.py

```diff
@@ -134,14 +137,14 @@
             "location_field_id": "field_1878",
             "service_id": "c3fd3bb177cc4291880bbe8c630ed5c4",
             "layer_id": 0,
-            "item_type": "layer"
+            "item_type": "layer",
         },
         "view_3003": {
             "description": "Signal detection status logs",
             "scene": "scene_514",
             "modified_date_field": "field_2565",
-            "socrata_resource_id": "e4b6-xseb"
-        }
+            "socrata_resource_id": "e4b6-xseb",
```

Comment:
I appreciate the `,`s after the last key-value pairs. Thanks for adding those.

Reply:
👍 linted via [Black](https://github.com/psf/black)